boards/seeed/seeeduino_xiao/seeed_xiao_connector.dtsi

```diff
@@ -30,3 +30,4 @@ xiao_spi: &sercom0 {};
 xiao_i2c: &sercom2 {};
 xiao_serial: &sercom4 {};
 xiao_dac: &dac0 {};
+xiao_adc: &adc {};
```

Comment:
You can remove:
> Needs the modification in PR https://github.com/zephyrproject-rtos/zephyr/pull/70683 in order to be able to use the
gcx01x driver without the reset-gpios property.

from the commit message and replace it with some information about this `xiao_adc` nodelabel. Or introduce this as part of separate commiit.

Reply:
I did the requested changes, thanks for the reviews :)